tests/Unit/Test_TestHelpers.cpp

```diff
@@ -25,3 +25,25 @@ SPECTRE_TEST_CASE("Test.TestHelpers", "[Unit]") {
   u_set.insert(0);
   test_iterators(u_set);
 }
+
+SPECTRE_TEST_CASE("Test.TestHelpers.Derivative", "[Unit]") {
+  const size_t VolumeDim = 3;
+
+  const std::array<double, VolumeDim> x{{1.2, -3.4, 1.3}};
+  const std::array<double, VolumeDim> dx{{1e-2, 0., 0.}}, dy{{0., 1e-2, 0.}},
+      dz{{0., 0., 1e-2}};
+
+  // We use a local approx value as opposed to the default 1e-14 since the
+  // numerical derivative won't be accurate to machine precision
+  Approx local_approx = Approx::custom().epsilon(1e-12);
+
+  auto func = [&](const std::array<double, VolumeDim>& x) {
```

Comment:
This isn't capturing anything, and the capture specification is confusing because of the two `x` variables.  Same for dfunc.